.eslintignore

```diff
@@ -1,6 +1,7 @@
 .vscode
 assets/
 bin/
```

Comment:
I think this needs to change to `bin/*` in order for the exception for `generate-proptypes-index` to be respected.